src/client/src/rt-platforms/browser/window.ts

```diff
@@ -44,9 +48,10 @@ export const openBrowserWindow = function(
   }
 
   this.prevWindow = win
+  this.externalWindows = this.externalWindows.filter(window => !window.closed).concat(win)
```

Comment:
Not your change ...but using `this` to hold the state is a bit confusing to me. Would be simpler IMO to define a couple of `let` vars outside of the function and use the closure.

Reply:
this was my change from a previous PR actually.  I removed the `this` binding to make the function more intuitive